build.gradle

```diff
@@ -119,3 +119,17 @@ jacocoTestReport {
         xml.required = true
     }
 }
+
+String version = '1.1.2'
```

Comment:
I probably missed something here but how can 1.1.2 be bumped to 1.1.3?
And how can we also bump minor version if needed?

Manual bump?

Reply:
Yes! When you create a feature or add a fix, example: Fixing the url path, bump this version in the same PR. The workflow will create an automated PR to increment version on 1.x. However, the user still needs to cut the tag manually.